front/src/store/modules/post.js

```diff
@@ -1,26 +1,38 @@
-import { MOCK_POSTS } from "../../utils/mock-data";
+import * as api from "../../api/posts";
 
 export default {
   namespaced: true,
   state: {
     posts: [],
   },
   mutations: {
+    ADD_POST(state, post) {
+      state.posts.push(post);
+    },
     SET_POSTS(state, posts) {
       state.posts = posts;
     },
-    ADD_POST(state, post) {
-      state.posts.push(post);
+    REMOVE_POST(state, postId) {
+      const deleteIndex = state.posts.findIndex((val) => val.id === postId);
```

Comment:
```suggestion
      const deleteIndex = state.posts.findIndex((post) => post.id === postId);
```

이건 어떨까요? 아니면 `item`이나 `element`도 괜찮을 것 같아요.

Reply:
아이코 수정했습니다!